dist/wtc-gallery-component.js

```diff
@@ -410,9 +434,12 @@ function (_ElementController) {
       }
 
       this.currentItem = next;
+      this.currentIndex = +next.dataset.index;
 
       if (this.options.autoplay) {
         this.player = setTimeout(this.next.bind(this), this.options.delay);
+      } else {
+        this.liveRegion.innerHTML = "Active carousel item: ".concat(this.currentIndex + 1, " of ").concat(this.items.length, ".");
```

Comment:
We should make this configurable. Maybe we need to have a language assets object for this class that has a default (as above) but allows us to pass different content for all of these areas.

Reply:
Added this in as well